packages/parser-less/tests/interface-tests.ts

```diff
@@ -0,0 +1,133 @@
+
+    return {
+        engine,
+        processor
+    };
+};
+
+test('If a style tag is not LESS, then nothing should happen', async (t) => {
+    const sandbox = sinon.createSandbox();
+    const element = mockStyleElement(null, '');
+    const resource = 'index.html';
+    const { engine, processor } = mockContext();
+
+    const postcssProcessSpy = sandbox.spy(processor, 'process');
+    const elementGetAttributeSpy = sandbox.spy(element, 'getAttribute');
+
+    await engine.emitAsync('element::style', { element, resource } as ElementFound);
+
+    t.true(elementGetAttributeSpy.calledOnce);
+    t.is(elementGetAttributeSpy.firstCall.args[0], 'rel');
+    t.false(postcssProcessSpy.called);
+
+    sandbox.restore();
+});
+
+test('If a style tag is inline LESS, then we should parse the stylesheet and emit a parse::end::css event', async (t) => {
+    const sandbox = sinon.createSandbox();
+    const code = '.foo { color: #fff }';
+    const element = mockStyleElement('stylesheet/less', code);
+    const resource = 'index.html';
+    const { engine, processor } = mockContext();
+
+    const engineEmitAsyncSpy = sandbox.spy(engine, 'emitAsync');
+    const postcssProcessSpy = sandbox.spy(processor, 'process');
+    const elementGetAttributeSpy = sandbox.spy(element, 'getAttribute');
+
+    await engine.emitAsync('element::style', { element, resource } as ElementFound);
+
+    t.true(elementGetAttributeSpy.calledOnce);
+    t.is(elementGetAttributeSpy.firstCall.args[0], 'rel');
+    t.true(postcssProcessSpy.calledOnce);
+    t.is(postcssProcessSpy.firstCall.args[0], code);
+    t.true(engineEmitAsyncSpy.calledThrice);
+
+    const args = engineEmitAsyncSpy.args[2];
+    const data = args[1] as StyleParse;
+
+    t.is(args[0], 'parse::end::css');
+    t.is(data.code, code);
+    t.is(data.element, element);
+    t.is(data.resource, resource);
+});
+
+test('If fetch::end::* is received, then we should check if it is LESS, parse the stylesheet, and emit a parse::end::css event', async (t) => {
+    const sandbox = sinon.createSandbox();
+    const { engine, processor } = mockContext();
+    const code = '.foo { color: #fff }';
+
+    const engineEmitAsyncSpy = sandbox.spy(engine, 'emitAsync');
+    const postcssProcessSpy = sandbox.spy(processor, 'process');
+
+    await engine.emitAsync('fetch::end::unknown', {
+        resource: 'styles.less',
+        response: {
+            body: { content: code },
+            mediaType: 'text/less'
+        }
+    } as FetchEnd);
+
+    t.true(postcssProcessSpy.calledOnce);
+    t.is(postcssProcessSpy.firstCall.args[0], code);
+    t.true(engineEmitAsyncSpy.calledThrice);
+
+    const args = engineEmitAsyncSpy.thirdCall.args;
+    const data = args[1] as StyleParse;
+
+    t.is(args[0], 'parse::end::css');
+    t.is(data.code, code);
+    t.is(data.element, null);
+    t.is(data.resource, 'styles.less');
+});
+
+test('If fetch::end::* is received for CSS, nothing should happen', async (t) => {
```

Comment:
Same comment as for SASS:

```suggestion
test('If fetch::end::css is received, nothing should happen', async (t) => {
```